### DISCUSSION TITLE
maxgraph npmjs organization created

### DISCUSSION THREAD

**Discussion by tbouffard** [maxGraph/maxGraph#65] [Development]
Hello
we haven't decided yet how we are going to publish maxgraph on npmjs.
To be able to publish using a [scope](https://docs.npmjs.com/about-organization-scopes-and-packages) (if we decide to do it), I have just created the maxgraph npmjs organization: https://www.npmjs.com/org/maxgraph.
I am the only member of this organization at the moment, please contact me to receive an invitation. 
If we finally decide that we don't need the organization, I will remove it later.

cc @mcyph @junsikshim @lreading @jgadsden @csouchet @coclav

**Comment by csouchet** [comment]
I'm ok to have a dedicated organization on npmjs 🙂

**Reply by junsikshim** [reply]
I don't have much experience regarding publishing on npmjs, so I'll follow whatever you guys decide on it :)

**Comment by coclav** [comment]
Great initiative. I take the "maxGraph" would be the TS / rewritten version ?

**Reply by tbouffard** [reply]
I think so: maxgraph/maxGraph, or maxgraph/maxgraph-core or whatever name
To be decided!

**Comment by haapjari** [comment]
Hello! Is there any update on publishing the package on npm? 

Thank you in advance! :)

**Reply by tbouffard** [reply]
@haapjari there is currently no update on this topic.
But yes, we should publish a first alpha version to let people test integration of maxgraph in their project.
I may start this topic on next week (without guarantee 😸 )

**Reply by haapjari** [reply]
Thank you very much on the update!

**Reply by tbouffard** [reply]
ℹ️ The package is not yet released but you can test it locally, see #87

**Comment by tbouffard** [comment]
ℹ️ If someone need to publish the maxgraph package in the maxgraph npm organization (https://www.npmjs.com/org/maxgraph), you can contact me or @csouchet.
